Docker/entrypoint.sh

```diff
@@ -24,6 +22,10 @@ if [ -n "$CRON_MIN" ]; then
 		-r "s#^[^ ]+ #$CRON_MIN #" | crontab -
 fi
 
+./cli/access-permissions.sh
```

Comment:
Access rights need to be ensured also before our CLI commands, in the case of external volumes with incompatible permissions.